templates/desktop/features.html

```diff
@@ -58,7 +58,7 @@ <h2>A whole world of apps</h2>
       <ul class="p-matrix is-trisected">
         <li class="p-matrix__item">
           <a href="https://snapcraft.io/spotify">
-            <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2017/12/spotify-linux-256.png" alt="Spotify icon" />
+            <img class="p-matrix__img" src="https://dashboard.snapcraft.io/site_media/appmedia/2017/12/spotify-linux-256.png?w=48" width="48" alt="Spotify icon" />
```

Comment:
dashboard.snapcraft.io doesn't seem to support the `?w=` parameter. There are a few instances in this file.
Two options:
- remove instances of the non-working parameter
- Bonus points: move this images to the asset server where they can be more efficiently served for this purpose.